pkg/extensions/sync/oci_layout.go

```diff
@@ -40,6 +40,9 @@ func (oci OciLayoutStorageImpl) GetContext() *types.SystemContext {
 
 func (oci OciLayoutStorageImpl) GetImageReference(repo string, reference string) (types.ImageReference, error) {
 	localImageStore := oci.storeController.GetImageStore(repo)
+	if localImageStore == nil {
+		return nil, fmt.Errorf("no image store found for repo %s", repo)
```

Comment:
Please define the new error in https://github.com/project-zot/zot/blob/main/errors/errors.go

Reply:
I've added a new error:

https://github.com/project-zot/zot/pull/2537/files#diff-d04adb084335cede75155605abdcc0bef92c45a740b274f985f378e955c81a13R90

I was not sure on the wording, so I reused another similarly sounding error. Feel free to suggest another wording.

I also added a simple test, since the CI was complaining about coverage :zany_face: